### ISSUE TITLE
JAVGG.net has no "next page" button

### ISSUE THREAD

**Issue by Hugo1290** [dobbelina/repository.dobbelina#1323]
### Prerequisites

- [X] I have searched the issue tracker to avoid opening a duplicate issue.
- [X] I have read the [documentation](https://github.com/dobbelina/repository.dobbelina/wiki/Cumination:-Debug-Logging-and-Uploading) on how to create a debug log file.

### Description

Hello Everyone,

there is no bug or error in the log, but i noticed, that the javgg.net site is missing its "next page" at the bottom of all lists, genres or actors list-pages. This occurs with kodi/cumination on windows as well as on android.

Basically i would like to know whether or not this problem is known and if by chance there is a fix.

Regards

### Kodi version

20.5 nexus 

### Cumination version

1.0.4 (auto-update)

### Operating System and version

Windows 11 / Android ARMV7A
